crates/turbopack-ecmascript/src/chunk/mod.rs

```diff
@@ -770,15 +771,14 @@ impl<'a> HmrUpdateEntry<'a> {
         struct Id<'a> {
             id: &'a ModuleId,
         }
+        let id = serde_qs::to_string(&Id { id: &entry.id }).unwrap();
         HmrUpdateEntry {
             code: entry.source_code(),
-            map: entry.code.has_source_map().then(|| {
-                format!(
-                    "/__turbopack_sourcemap__/{}.map?{}",
-                    chunk_path,
-                    serde_qs::to_string(&Id { id: &entry.id }).unwrap()
-                )
-            }),
+            url: format!("/{}?{}", chunk_path, &id),
```

Comment:
Since this url creates a visible entry in the dev tools sources which is accessible via Ctrl+p, let's hash it for the url to avoid users seeing multiple options for they filename when typing into Ctrl+p

Reply:
I tried it and it's fine since due to the urlencoding it's very difficult to confuse